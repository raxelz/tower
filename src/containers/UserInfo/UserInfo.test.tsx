import { shallow } from 'enzyme';
import * as React from 'react';
import { UserInfo } from './';

const defaultProps = {
    match: {
        params: {
            uid: '',
        },
    },
};

describe('UserInfo test', () => {
    it('should render', () => {
        const wrapper = shallow(<UserInfo {...defaultProps} />);
        expect(wrapper).toMatchSnapshot();
        expect(wrapper).toBeDefined();
    });
});
