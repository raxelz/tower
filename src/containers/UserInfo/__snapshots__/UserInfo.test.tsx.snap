// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UserInfo test should render 1`] = `
<ContextConsumer>
  <Component />
</ContextConsumer>
`;
